pydomcfg/utils.py

```diff
@@ -80,7 +82,12 @@ def _calc_rmax(depth: DataArray) -> DataArray:
     return rmax.fillna(0)
 
 
-def _smooth_MB06(depth: DataArray, rmax: float) -> DataArray:
+def _smooth_MB06(
+    depth: DataArray,
+    rmax: float,
+    tol: float = 1.0e-8,
+    max_iter: int = 10_000,
+) -> DataArray:
     """
     This is NEMO implementation of the direct iterative method
```

Comment:
I would add something like: This is the xarray version of NEMO implementation ....

Reply:
Reworded, although I didn't explicitly mention xarray as we use it pretty much everywhere and it should be clear from the docs. Also I removed the units, any units should work fine.

Reply:
Yes, I agree with your point ... it was just to make clear to the users (an us :) ) that this is only a more efficient version of the NEMO MB06 implementation